helm-chart/templates/prometheus.yaml

```diff
@@ -32,65 +32,28 @@ metadata:
     prometheus: {{ .Values.prometheus.selector }}
 spec:
   groups:
-    {{ if .Values.prometheus.alerts.panics.useDefault }}
-    - name: ScaleToZeroPanic
+    {{ range .Values.prometheus.alertGroups }}
```

Comment:
Named template in range makes it easier to read, 
```
 {{ range $alertGroup := .Values.prometheus.alertGroups }}
- name: {{ $alertGroup.name }}
-   {{ range $alert := $alertGroups.alerts }}
        - alert: {{ $alert.name}}
          expr: {{ $alert.expression }}
          for: {{ $alert.for }}
```

Reply:
yep, updated